src/plots/cartesian/axes.js

```diff
@@ -2063,8 +2065,11 @@ function num2frac(num) {
 // also automatically switch to sci. notation
 var SIPREFIXES = ['f', 'p', 'n', 'μ', 'm', '', 'k', 'M', 'G', 'T'];
 
+// extending SI prefixes
+var SIPREFIXES_EXTENDED = ['q', 'r', 'y', 'z', 'a', 'f', 'p', 'n', 'μ', 'm', '', 'k', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y', 'R', 'Q'];
+
 function isSIFormat(exponentFormat) {
-    return exponentFormat === 'SI' || exponentFormat === 'B';
+    return exponentFormat === 'SI' || exponentFormat === 'SI extended' || exponentFormat === 'B';
 }
```

Comment:
You could turn this into a one-liner:

```suggestion
const isSIFormat = (exponentFormat) => ['B', 'SI', 'SI extended'].includes(exponentFormat);
```